ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceWithPredefinedOperatorFix.fs

```diff
@@ -0,0 +1,37 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+
+type ReplaceWithPredefinedOperatorFix(error: AddingConstraintError) =
+    inherit FSharpQuickFixBase()
+
+    let ref = error.Expr.As<IReferenceExpr>()
+
+    let fsharpOperator =
```

Comment:
What does `fsharp` stand for here?

Reply:
renaming lost before submitting the commit :(